Brandon_hopper.asm

```diff
@@ -0,0 +1,228 @@
+# COMP3410 Program Template
+# Author: Brandon Hopper
+# Assignment: PA[X]
+# Date: 02/11/2015
+
```

Comment:
*Please separate each program into its own file.  Generally if a program doesn't compile its automatically worth 0 points.  I separated them this time but I can't do that for additional assignments.